package.json

```diff
@@ -27,20 +29,23 @@
     "prop-types": "^15.6.2",
     "ramda": "^0.26.0",
     "raven-js": "^3.27.0",
-    "react": "^16.6.3",
+    "react": "^16.9.0",
     "react-datetime-picker": "^2.4.0",
-    "react-dom": "^16.6.3",
+    "react-dom": "^16.9.0",
     "react-ga": "2.5.3",
+    "react-hot-loader": "^4.12.14",
     "react-markdown": "^4.0.3",
     "react-mde": "^7.5.0",
     "react-redux": "^5.1.1",
-    "react-router-dom": "^4.3.1",
+    "react-router": "^5.1.2",
```

Comment:
i believe `react-router-dom` [re-exports all of `react-router`'s core component](https://github.com/ReactTraining/react-router/issues/4648#issuecomment-284479720), so we should only have to install the one. though i see you added it recently so maybe you were running into problems not having it?